index.js

```diff
@@ -514,6 +533,9 @@ const run = async userOptions => {
         cacheAjaxRequests,
         preconnectThirdParty
       } = options;
+
+      if (!route.includes("/404.html")) sitemapItems.push(route)
```

Comment:
```js
route.endsWith("/404.html")
```

Also I'm trying to figure out alternative ways to detect 404 pages:

```js
if (title.includes("404")) console.log(`⚠️  page not found ${route}`);
```

but I do not like this approach